services/headless-lms/server/src/controllers/course_material/pages.rs

```diff
@@ -18,6 +18,17 @@ async fn get_by_exam_id(
     let token = skip_authorize()?;
     token.authorized_ok(web::Json(page))
 }
+/**
+GET /api/v0/course-material/pages/exam/{page_id}
```

Comment:
```suggestion
GET /api/v0/course-material/courses/{course_id}/top-level-pages
```

Also move to courses.rs